Doc/library/stdtypes.rst

```diff
@@ -5492,6 +5492,17 @@ types, where they are relevant.  Some of these are not reported by the
    The tuple of base classes of a class object.
 
 
+.. attribute:: definition.__module__
```

Comment:
In my opinion, the attributes listed here should be attributes that appear on _all_ Python objects (I'm aware that's not currently the case, but that's a problem, in my opinion).

We already have a dedicated reference for the `__module__` attribute on function objects; I think we should do the same for the `__module__` attribute on class objects: https://docs.python.org/3/reference/datamodel.html#function.__module__ (ditto for `__doc__`)